logistics.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title: Get ready for the first virtual OHBM Hackathon!
+isStaticPost: true
+permalink: /logistics/
+---
+
+<img align="right" src="../img/undraw_connected_world_wuay.svg" alt="connected_world" width="40%">
+
+#### OHBM Hackathon Hubs
+
+{% accordion hackathonHubs %}
+{% collapse What are the hubs? %} 
+In addition to this event being the first ever virtual OHBM hackathon, it will also be happening three times within a 24 hour window, across multiple time-zones. The goal here is to ensure that more people from around the world are able to join in during reasonable working hours for them. 
+* Americas Hub - Hosts: Liza Levitis 
+* Europe, Middle East, and Africa Hub - Hosts: Remi Gau
+* Asia and Pacific Oceania Hub - Hosts: Angela Tam and Chao-Gan Yan
+{% endcollapse %}
+
+{% collapse Hub structure and general organization %} 
+In order for the event to run as smoothly as possible across all hubs, each hub will have a list of roles filled by volunteers. 
+Some of the roles are selected by the OS-SIG while others are open to volunteers.
+
+##### Selected by the OS-SIG
```

Comment:
Do we want to mention which ones are selected by us or do we only want to have those we people can "volunteer" for?